src/main/frontend/pipeline-console-view/pipeline-console/main/StageNodeLink.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export interface StageNodeLinkProps {
+    agent: string;
+}
+
+const StageNodeLink = ({agent}: StageNodeLinkProps) => {
+    const agentName = agent == "built-in" ? "Jenkins" : agent;
+    const href = `../../../../computer/${agent == "built-in" ? "(built-in)" : agent }/`;
```

Comment:
I don't think this will be reliable, you could be nested in multi-level folders. you should do it from the root, you can access the relative URL that Jenkins is running on from with (in plain JS, react will be a little different): `document.head.dataset.rooturl`

roughly:
`${document.head.dataset.rooturl}/manage/computer/...`

but you may need a react hook to get the dataset

Reply:
I've updated the code to use the root url. Could not really find a reason why a hook would be needed, so I hope it's good.